src/main/kotlin/racingcar/Racing.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar
+
+class Racing(
+    private val trials: Int,
+    private val cars: Collection<Car>
+) {
+    init {
+        if (cars.size < 2) {
+            throw IllegalArgumentException("At least two cars must be participated. count='${cars.size}'")
+        }
```

Comment:
`require()`는 값을 만족하지 않으면 `IllegalArgumentException`을 발생시킵니다.

```suggestion
        require(cars.size >= 2) { "At least two cars must be participated. count='${cars.size}'" }
```